packages/react/src/dialog/Dialog.css.ts

```diff
@@ -0,0 +1,65 @@
+import { theme } from "../styles";
+import { type RecipeVariants, recipe, style } from "../vanilla-extract";
+
+export const content = recipe({
+  base: [
+    {
+      bg: "white",
+      rounded: "lg",
+      shadow: "md",
+    },
+    style({
+      position: "relative",
+    }),
+  ],
+  variants: {
+    size: {
+      sm: style({
+        maxWidth: "90dvw",
+        width: "375px",
+      }),
+      md: style({
+        maxWidth: "90dvw",
+        width: "600px",
+      }),
+      lg: style({
+        maxWidth: "90dvw",
+        width: "800px",
+      }),
+    },
+  },
+});
+
+export const overlay = recipe({
+  base: [
+    {
+      alignItems: "center",
+      bg: "dark.200",
+      size: "full",
+    },
+    style({
+      position: "absolute",
+    }),
+  ],
+});
+
+export const close = recipe({
+  base: [
+    {
+      p: "2",
+      rounded: "full",
+      size: "20",
```

Comment:
let's remove the padding and sizing since `<Button />` will take care of that
```suggestion
      rounded: "full",
```